package.json

```diff
@@ -23,38 +23,29 @@
     "to-camel-case": "^1.0.0"
   },
   "engines": {
-    "node": ">=5.10.0"
+    "node": ">=16"
```

Comment:
why this and on >=14 ? 
if we're gonna break, break it good?

Reply:
Oh, that's because 16 is the lowest LTS. I can change it to 14.

Reply:
I just tried to use 14 in CI, and it didn't work because we use `??=`

```
timeoutInactivity ??= opts.timers ? 600e3 : 5e3
```

So it's indeed >=16.